src/main/scala/com/databricks/labs/overwatch/pipeline/PipelineTargets.scala

```diff
@@ -154,6 +154,53 @@ abstract class PipelineTargets(config: Config) {
       config
     )
 
+    lazy private[overwatch] val libsSnapshotTarget: PipelineTable = PipelineTable(
+      name = "libs_snapshot_bronze",
+      _keys = Array("cluster_id", "Overwatch_RunID"),
+      config,
+      incrementalColumns = Array("Pipeline_SnapTS"),
+      partitionBy = Seq("organization_id")
+    )
+
+    lazy private[overwatch] val policiesSnapshotTarget: PipelineTable = PipelineTable(
+      name = "policies_snapshot_bronze",
+      _keys = Array("policy_id", "Overwatch_RunID"),
+      config,
+      incrementalColumns = Array("Pipeline_SnapTS"),
+      partitionBy = Seq("organization_id")
+    )
+
+    lazy private[overwatch] val instanceProfileSnapshotTarget: PipelineTable = PipelineTable(
+      name = "instance_profile_snapshot_bronze",
+      _keys = Array("cluster_id", "Overwatch_RunID"),
+      config,
+      incrementalColumns = Array("Pipeline_SnapTS"),
+      partitionBy = Seq("organization_id")
+    )
+
+    lazy private[overwatch] val tokenSnapshotTarget: PipelineTable = PipelineTable(
+      name = "token_snapshot_bronze",
```

Comment:
please convert all single asset names to plural to maintain uniformity throughout bronze. This name should be tokens_snapshot_bronze instead of token.

instance_profile should be instance_profiles... etc. Notice how even in this change you have some plural (i.e. libs) and some singular. Make uniform to plural